docs/core/extensions/dependency-injection.md

```diff
@@ -63,9 +63,19 @@ This interface is implemented by a concrete type, `MessageWriter`:
 
 The sample code registers the `IMessageWriter` service with the concrete type `MessageWriter`. The <xref:Microsoft.Extensions.DependencyInjection.ServiceCollectionServiceExtensions.AddScoped%2A> method registers the service with a scoped lifetime, the lifetime of a single request. [Service lifetimes](#service-lifetimes) are described later in this article.
 
-:::code language="csharp" source="snippets/configuration/dependency-injection/Program.cs" highlight="16":::
+:::code language="csharp" source="snippets/configuration/dependency-injection/Program.cs" highlight="5-8":::
 
-In the sample app, the `IMessageWriter` service is requested and used to call the `Write` method:
+In the preceding code, the sample app:
+
+- Creates a host builder instance.
+- Configures the services, which registers:
+
+  - The `Worker` as a hosted service. For more information, see [Worker Services in .NET](workers.md).
+  - The `IMessageWriter` interface as a scoped service with a corresponding implementation of the `MessageWriter` class.
+
+- The builder builds the host and runs it.
```

Comment:
```suggestion
- Builds the host and runs it.
```

Reply:
Otherwise it doesn't match up with the intro text, i.e., "The sample app the builder builds the host..."